package.json

```diff
@@ -141,9 +141,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "eslint \"src/**/*.ts\" \"test/**/*.ts\" --fix; if git --no-pager diff HEAD package.json | egrep '\"version\": \"[0-9]+\\.'; then npm run version; else (npm --no-git-tag-version version patch; git add package.json; sf-husky-pre-commit); fi;",
-            "commit-msg": "sf-husky-commit-msg",
-            "pre-push": "sf-husky-pre-push"
+            "pre-commit": "eslint \"src/**/*.ts\" \"test/**/*.ts\" --fix; if git --no-pager diff HEAD package.json | egrep '\"version\": \"[0-9]+\\.'; then npm run version; else (npm --no-git-tag-version version patch; git add package.json; sf-husky-pre-commit); fi;"
```

Comment:
Should we have two separate places where we have `pre-commit` logic? I think if possible we should put this in the `./husky/pre-commit` file. We should also probably remove the `sf-husky-pre-commit` line since we're using regular `husky` and not `sf-husky`